src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,17 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { Person } from '../../type';
+
+type Props = {
+  person: Person,
+};
+
+export const UserInfo: React.FC<Props> = ({ person }) => {
+  return (
```

Comment:
I guess props destructuring would be a good idea here as well
p.s. And I suppose some tests are failing because of naming - person instead of user.